builds/e2e/templates/longhaul-deploy.yaml

```diff
@@ -19,14 +19,14 @@ steps:
   - task: CopyFiles@2
     displayName: 'Copy Edgelet Artifact'
     inputs:
-      SourceFolder: '$(Build.StagingDirectory)/$(edgelet.artifact.name)'
-      TargetFolder: '$(Agent.HomeDirectory)/../artifacts/$(edgelet.artifact.name)'
+      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['edgelet.artifact.name'] }}"
```

Comment:
changes in the file is to make consistent syntax in long haul and stress test template; copied from stress test template, which use parameter[] is safer since variable name contains period.